src/test/resources/compiler/choice_obj_values.wdl

```diff
@@ -0,0 +1,38 @@
+version 1.0
+
+# adding a choices attr to the parameter_meta section for an input variable
+# should propogate that var to the interface
+
+# Correct
+task choice_values_cgrep {
+    input {
+        String pattern
+        File in_file
+    }
+    parameter_meta {
+        in_file: {
+          choices: [
+            {
+              name: "file1", 
+              value: "dx://file-Fg5PgBQ0ffP7B8bg3xqB115G"
+            },
+            {
+              name: "file2", 
+              value: "dx://file-Fg5PgBj0ffPP0Jjv3zfv0yxq"
+            }
+          ]
+        }
+        pattern: {
```

Comment:
Here and in the other examples. I probably have a misunderstanding. Isn't the `pattern` scoped to the `in_file` variable?

Reply:
See above